src/helpers/ckEditorConfig.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+
+// Function That will return a promise object when resolved
+export const getImageUrl = (file) => new Promise((resolve, reject) => {
```

Comment:
kindly change this `getImageUrl` to use `es6` pattern of `async/await` instead of `new Promise`